### ISSUE TITLE
How do I resolve non-Mason dependencies

### ISSUE THREAD

**Issue by buddha314** [chapel-lang/chapel#8771]
Trying to run `mason build` on https://github.com/buddha314/numsuch/releases/tag/v0.2.2 I get

```
 mason build
Updating mason-registry for local_registry
Updating mason-registry for mason-registry
Compiling NumSuch
In file included from /var/folders/g5/4zwyfk513hd94tjs1fm5mdq00000gn/T//chpl-buddha-55353.deleteme/_main.c:2:
/var/folders/g5/4zwyfk513hd94tjs1fm5mdq00000gn/T//chpl-buddha-55353.deleteme/chpl__header.h:5:10: fatal error: 'cblas.h' file not found
#include "cblas.h"
         ^~~~~~~~~
1 error generated.
make: *** [/var/folders/g5/4zwyfk513hd94tjs1fm5mdq00000gn/T//chpl-buddha-55353.deleteme/NumSuch.tmp] Error 1
error: compiling generated source
Build Failed
```

I'm not sure how to use Mason to tell the compiler my `BLAS_HOME= /usr/local/Cellar/openblas/0.2.20`

**Comment by buddha314** [comment]
I expect the answer to how to handle this will be addressed somewhere in this issue: https://github.com/chapel-lang/chapel/issues/7106 

The notion of `.COMPOPTS` in the testing suite might not be a bad way to go.

**Comment by buddha314** [comment]
I have experimented with `cmake` and [`find_package`](https://cmake.org/cmake/help/v3.0/command/find_package.html) a bit. I can't quite get it to work, but perhaps if we did a mason installer and it installed according to the `cmake` guidelines, it could be cobbled together.  Oh, looks this ticket touches on that: https://github.com/chapel-lang/chapel/issues/7639

**Comment by ben-albrecht** [comment]
This is a planned feature for Mason.

**Comment by Spartee** [comment]
More to come with this in #10260 but this ability has been merged into mason with #10289
